src/DotNetWorker/Converters/Converter/DefaultConverterContext.cs

```diff
@@ -19,6 +19,6 @@ public DefaultConverterContext(FunctionParameter parameter, object? source, Func
 
         public override FunctionParameter Parameter { get; set; }
 
-        public override FunctionExecutionContext ExecutionContext { get; set; }
+        public override FunctionContext ExecutionContext { get; set; }
```

Comment:
Should this property be called `Context` instead of `ExecutionContext`.

Reply:
Good catch! Will update.